BH/MPQInit.cpp

```diff
@@ -293,8 +293,8 @@ ItemAttributes ItemAttributeList[] = {
 	{"Chipped Diamond", "gcw", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 65, 0},
 	{"Flawed Diamond", "gfw", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 66, 0},
 	{"Diamond", "gsw", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 68, 0},
-	{"Flawless Diamond", "glws", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 72, 0},
-	{"Perfect Diamond", "gpws", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 80, 0},
+	{"Flawless Diamond", "glw", "Gem", 1, 1, 0, 0, 0, 0, 0, 0, 72, 0},
```

Comment:
Why did you change diamonds to use the unstacked code? It seems like every other flawless/perfect gem in this set is using the stacked code

Reply:
The first group (268-297) is the non-stacked ones and it looks like these two were changed to use the stacked codes. Row 308 has the stacked Flawless Diamond, although I'm just now realizing that stacked Perfect Diamond is missing. I'll add it